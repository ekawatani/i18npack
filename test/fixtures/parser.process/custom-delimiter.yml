foo: "<%= bar %>"
bar: This is a test.
