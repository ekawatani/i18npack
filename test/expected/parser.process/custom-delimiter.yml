{
  "foo": "This is a test.",
  "bar": "This is a test."
}